.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "esbonio.sphinx.confDir": "${workspaceFolder}/copter/source",
+    "grammarly.config.documentDialect": "american",
+    "editor.wordWrap": "on",
```

Comment:
this seems like out of scope of this PR change, some devs might not want this.

Reply:
Good catch! I tend to keep this on for the wiki, but not everyone likes word wrapping.